### ISSUE TITLE
Support combining identities

### ISSUE THREAD

**Issue by lrettig** [spacemeshos/pm#267]
In #253 we're planning to allow smeshers to increase and decrease their identity size. But this doesn't solve the issue of smeshers who already have multiple small identities that they want to combine into one identity. This will matter more when we begin incentivizing larger ATXs/implementing ATX penalties (#266).

For more see:
- [“Naïve” ATX combining](https://community.spacemesh.io/t/naive-atx-combining/376) (private)

**Comment by dshulyak** [comment]
i don't understand why opened it, lets please finalize research discussion, and if it proves practical it will be added in go-spacemesh

**Comment by dshulyak** [comment]
set to low priority as current best solution solves very little, see research discussion for details

**Comment by dshulyak** [comment]
link to the solution https://community.spacemesh.io/t/naive-atx-combining/376

solving this particular issue will ensure that we won't have problems with consensus due to bandwidth blowup. 
it will also help to address issues with non-unique activesets, and slightly reduce amount of memory that node needs.

it won't be a small task, as it will require special malfeasence proof and new atx version. will probably take 2-4 weeks.
and likely can be completed only after #261

**Comment by mathcrypto** [comment]
See the research issue on the topic https://github.com/spacemeshos/pm/issues/297
